### PR TITLE
Graduate amp-social-share out of experiments

### PR DESCRIPTION
**Pull request by mkhatib**
Do not submit until next validator release is deployed!

### CODE REVIEW

**Comment by erwinmombay** [comment]
@mkhatib please update markdown docs and mark with documentation label as well

File: tools/experiments/experiments.js [hunk 1]
```diff
@@ -55,12 +55,6 @@ const EXPERIMENTS = [
     name: 'Activates support for measuring incoming clicks.',
     spec: 'https://github.com/ampproject/amphtml/issues/2934',
   },
-  {
```
Comment (dvoytenko) [inline tools/experiments/experiments.js]: Please also update `.md` file. It should say "experimental" somewhere there.

**Comment by mkhatib** [comment]
> mark with documentation label as well

@erwinmombay not sure what did you mean with this part?

Updated `.md` file

File: tools/experiments/experiments.js (hunk 1 above)
Comment (mkhatib) [inline tools/experiments/experiments.js]: Done.

**Comment by mkhatib** [comment]
`amp-social-share` validation rules should be already deployed! @dvoytenko can you PTAL this?

**Comment by dvoytenko** [comment]
LGTM

**Comment by aoishailesh** [comment]
@mkhatib @erwinmombay - I was just about to start a PR when I saw this commit - thanks so much for taking care of this so quickly!

**Comment by erwinmombay** [comment]
forced to be out of experimental in b/123049011

**Pull request merged**